package.json

```diff
@@ -2,7 +2,7 @@
   "name": "bcros-business-dashboard",
   "private": true,
   "type": "module",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Please rebase again. The version is already 1.0.2.

https://github.com/bcgov/business-dashboard-ui/blob/main/package.json